src/latest_release_note.ts

```diff
@@ -1,28 +1,26 @@
 // TODO: find a better way of loading html into a string
-export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes 🐍📝 (Mar. 10, 2020)</h1>
+export const LATEST_RELEASE_NOTE = `<h1>Device Simulator Express Release Notes 🍰🐍 (Mar. 24, 2020)</h1>
 <p>
-    Thanks for using our extension! We're back with some small updates. <u>Again, please feel free to enable our feature
-        flag in
-        Settings
-        (under the setting titled “<b>deviceSimulatorExpress.previewMode</b>” in the User settings)</u>.
-</p>
+    Thanks for using our extension! <b>Support for micro:bit simulation is now officially released on the
+        Device Simulator Express</b>. 💖✨</p>
 <h2>Changes</h2>
```

Comment:
Perhaps remove the "Changes" header, since it's redundant (we already have fixes/features below it).